paket.dependencies

```diff
@@ -2,20 +2,20 @@ source https://nuget.org/api/v2
 
 nuget Newtonsoft.Json redirects: force
 nuget Argu prerelease
-nuget FSharp.Core prerelease redirects: force
+nuget FSharp.Core == 4.0.1.7-alpha redirects: force
```

Comment:
This is the same as for the test library, which ends up referencing the projects using FSharp.Core from outside of the test group so it has to be the same version in both of them, it just says it up top now instead of down below.

`4.0.0.1` doesn't support netcore so it's either this or we add a new group for netcore switch this to `4.0.0.1` and use `4.1.0` with netcore. 

It's just a superficial change.